pkgs/applications/networking/owncloud-client/libre-graph-api-cpp-qt-client.nix

```diff
@@ -19,17 +18,9 @@ stdenv.mkDerivation rec {
 
   sourceRoot = "${src.name}/client";
 
-  nativeBuildInputs = [ cmake wrapQtAppsHook ];
+  nativeBuildInputs = [ cmake ];
   buildInputs = [ qtbase ];
 
+  dontWrapQtApps = true;
   cmakeFlags = [  ];
-
-  meta = with lib; {
```

Comment:
Why delete meta?

Reply:
if it's internal library, is meta relevant?

Reply:
Yes, meta is very relevant and everything that uses mkDerivation should have meta.